replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+
+(4.2.1) A quick check of N by gender in data:
+
+```{r echo = FALSE}
+
+#Get counts and check values against article for correct coding scheme:
+count(dat_egood, geslacht1)
+#confirmed:
+## 1 = male
+## 2 = female
+
+```
+
+(4.2.2) Code for logistical regressions:
+
+```{r eval = FALSE}
+
+#Recode dummy variable for music as 0 and 1 for easier interpretation of estimates:
+Music <- factor(dat_egood$music,
+                    levels = c(1,2),
+                    labels = c("LikedMusic", "DislikedMusic"))
+
+dat_egood$music <- as.numeric(dat_egood$music)
+dat_egood <- dat_egood %>%
+  mutate(music.recode2 = (music - 1))
+View(dat_egood)
+
+#Set factor names:
+Music <- factor(dat_egood$music.recode2,
+                    levels = c(0, 1),
+                    labels = c("Liked", "DisLiked"))
+
+ChosenPen <- factor(dat_egood$chose_advertised_pen,
+                levels = c(1, 0),
+                labels = c("ChoseAdPen", "NotChoseAdPen"))
+
+#Compare counts of each gender in paper to data for coding:
+count(dat_egood, geslacht1)
+## 1 = male
+## 2 = female
+
+# need to check which is which:
+Gender <- factor(dat_egood$geslacht1,
+                    levels = c(1,2),
+                    labels = c("Male", "Female"))
+
+# for relationship between advertised pen and music, music 2 (disliked) seems significant:
+fit1 <- glm(ChosenPen ~ Music, data = dat_egood, family = binomial)
+fit1 <- tidy(fit1)
+fit1
+
+#### calculate odds 
+fit1$estimate
+fit1$estimate[1] + fit1$estimate[2] ## 1.216
+## odds ratio
+exp(fit1$estimate[2]) ##4.43
+## predicted probabilities:
+# Disliked Music:
+(exp(fit1$estimate[1] + fit1$estimate[2])) / (1 + exp(fit1$estimate[1] + fit1$estimate[2]))
+# Liked Music: 
+exp(fit1$estimate[1]) / (1 + exp(fit1$estimate[1])) 
+
+# does the relationship change when looking ONLY at males & ONLY females?
+
+male.dat_egood <- dat_egood %>%
+  filter(geslacht1 == "1")
+#count(male.dat_egood, geslacht1)
+
+female.dat_egood <- dat_egood %>%
+  filter(geslacht1 == "2")
+#count(female.dat_egood, geslacht1)
+
+# MALES:
+fitmale <- glm(ChosenPen ~ Music, data = male.dat_egood, family=binomial)
+fitmale <- tidy(fitmale)
+fitmale
+#### calculate odds 
+fitmale$estimate
+fitmale$estimate[1] + fitmale$estimate[2] ## 1.216
```

Comment:
It seems unnecessary to report the output in a comment. Couldn't you just return the output. In the spirit of transparency, I would trust what you were saying more if I could actually see the output.